docs/administration/high-volume-log-loss.adoc

```diff
@@ -0,0 +1,362 @@
+
+Normal operation (no log loss)::
+Send-rate exceeds write-rate. Logs are forwarded as quickly as they arrive.
+The set of uncollected logs is small and does not grow significantly.
+The forwarder has time to collect all logs before they are rotated.
+
+Overload (possible log loss)::
+Write-rate exceeds send-rate (on average) for an extended period of time.
+This could be caused by high write activity and/or slow sending.
+Uncollected logs accumulate as the forwarder falls behind.
+A brief overload (sometimes called a _burst_ or _spike_) can be handled without log loss if there is enough storage.
+An overload that continues long enough force rotation of uncollected logs **will** cause log loss.
+
+Recovery (no log loss, but not yet back to normal)::
+After an overload, there are more uncollected logs accumulated in storage than during normal operation.
+We must consider the time it takes to reduce this backlog and get back to normal, before the next overload.
+
+IMPORTANT: It is theoretically impossible to prevent log loss under all conditions.
+The goal is to analyze your expected average and peak conditions, then provision storage accordingly.
+
+=== Calculations and examples
+
+*Time to log loss during steady overload:*
+
+When write rate consistently exceeds send rate, logs accumulate until storage fills:
+
+```
+Time to log loss = Storage capacity ÷ (Write rate - Send rate)
+```
+
+*Example - Steady overload:*
+- Storage capacity: 500MB (kubelet default: 10MB × 5 files × 10 containers)
+- Write rate: 2MB/s
+- Send rate: 1.5MB/s
+- Net accumulation: 0.5MB/s
+- Time to log loss: `500MB ÷ 0.5MB/s = 1000 seconds (16.7 minutes)`
+
+*Example - Burst overload:*
+- Storage capacity: 500MB
+- Normal write rate: 1MB/s
+- Burst write rate: 5MB/s for 3 minutes
+- Send rate: 1.5MB/s (constant)
+- Logs accumulated during burst: `(5-1.5)MB/s × 180s = 630MB`
+- Result: **Log loss occurs** - burst exceeds storage capacity
+
+These examples show why understanding your specific rates and durations is critical for proper storage sizing.
+
+*Recovery time calculation:*
+
+After a peak logging period, the system needs time to drain accumulated logs. Calculate recovery time:
+```
+Recovery_time = Accumulated_logs / (Send_rate - Normal_log_rate)
+```
+
+*Key considerations:*
+
+- **Overlapping peaks:** If peaks occur closer together than recovery time, logs accumulate across multiple peaks
+- **Send rate degradation:** Network issues during recovery extend drain time
+- **Grace period:** Allow buffer between peaks for unexpected delays
+
+*Example recovery calculation:*
+- Peak accumulated: 1GB of logs
+- Normal log rate: 100 KB/s
+- Send rate: 400 KB/s
+- Recovery time: `1GB / (400-100) KB/s = 1GB / 300KB/s = 3,413 seconds (57 minutes)`
+
+If peaks occur more frequently than 57 minutes, you need larger storage or higher send capacity.
+
+== Recommendations
+
+The core recommendation is to _increase the log rotation limits of Kubelet_ based on
```

Comment:
IMO we should emphasize this with one of those boxes or other to make it stand out